packages/ckeditor5-html-support/tests/htmlcomment-integration.js

```diff
@@ -969,9 +969,7 @@ describe( 'HtmlComment integration', () => {
 
 			editor.execute( 'delete' );
 
-			// Currently, comments inside a deleted content are moved to adjacent paragraph.
-			// See https://github.com/ckeditor/ckeditor5/issues/10119.
-			//
+			// The following output will be also correct:
```

Comment:
I don't get it: when the following output will be also correct? 🤔

Reply:
TBH for me, both behaviors would be correct here and I'm not sure what should be the expected behavior.

But perhaps I can remove it.

Reply:
Currently, the `comment 4` is at the boundary of the removed range. So as the whole paragraph is removed, you could expect that the comment will be removed too.

Reply:
I got your point, but IMHO it is confusing now. I think that this sentence should be either changed to include i.e. why this output is not valid currently (and `comment 4` is not removed), or just remove it.

Something like: _The following output could be considered as the correct and expected one, but currently the `comment 4` is not removed, because it is not located at the limit element's boundary:_